### ISSUE TITLE
AVT 1 - Select Icon has DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4222]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
Go to the React Form Component 
Run DAP and the following violations are present (see screenshot):
<img width="1161" alt="Screen Shot 2019-10-04 at 9 08 15 AM" src="https://user-images.githubusercontent.com/21676914/66214214-ed22d580-e686-11e9-8044-f05d520c5b1d.png">

**Comment by joshblack** [comment]
Looking into this, seems like it's an issue on Select with the underlying `<svg>` using `role="img"`. `Form` itself is a wrapper and so I'll move this under that milestone and rename as appropriate.

**Comment by abbeyhrt** [comment]
@joshblack when you looked into it, were you able to recreate the DAP violation? I haven't been able to on the Select component or the Form

**Comment by joshblack** [comment]
@abbeyhrt I believe the issue is caused by this markup:

![image](https://user-images.githubusercontent.com/3901764/69440221-069ee000-0d0e-11ea-8ce0-61a1f1a1db7a.png)

Viewable at: http://react.carbondesignsystem.com/?path=/story/form--default
